src/protocols/fp-file.ts

```diff
@@ -0,0 +1,144 @@
+import { FPEnvelopeTurnaround, FPEnvelopeType, FPStoreType } from "./fp-envelope";
+import { CID } from "multiformats";
+
+export interface FPGetFileReq {
+    readonly cid: CID;
+    readonly fileName?: string;
+}
+/*
+ * A request to get a FILE file.
+ * As JSON, this is:
+ * {
+ *    "type": "get-file-req",
+ *    "tid": "123",
+ *    "payload": {
+ *       "cid": "bafybeib2v"
+ *    }
+ *    "auth": { ... }
+ *    "meta": { ... }
+ * }
+ */
```

Comment:
we need to be able to serve unencrpyted files directly at eg: `<img src="https://example.fireproof.storage/database-uuid/document-uuid/file.jpg"/>`

Reply:
being able to PUT raw files is not required